packages/patrol_cli/lib/src/runner/patrol_command.dart

```diff
@@ -43,7 +43,6 @@ abstract class PatrolCommand extends Command<int> {
       ..addFlag(
         'debug',
         help: 'Build a debug version of your app (default mode)',
-        defaultsTo: true,
```

Comment:
Won't that remove info about this from help? How are you telling the user that debug is a default?

Reply:
> Build a debug version of your app (default mode)

I know it's not perfect but works.